### PR TITLE
Remove tableify and rimraf

### PR DESCRIPTION
**Pull request by charisprose**

### CODE REVIEW

File: dash/backend/src/modules/auth/controllers/non-redirectable-login.controller.ts [hunk 1]
```diff
@@ -103,6 +104,14 @@ export class NonRedirectableLoginController {
     async localAuthLoginAction(@Body() loginDto: LoginDto): Promise<any> {
         this.prometheusV1Service.loginCounter.inc(1);
         const users: UserProfileDto[] = await this.userProfileService.loadUserByEmail(loginDto.userName);
+        // array
+        const sampleJson = ['banana', 'apple', 'peach'];
+        // message object const sampleJson = {label: 'Error creating webhook for cluster', data: { clusterId: 5 }};
+        // null const sampleJson = null;
+        // text with symbols const sampleJson = '09:36:29.597260967: Notice Unexpected connection to K8s API Server from container (command=python -u /app/sidecar.py pid=46515 k8s.ns=monitoring-system k8s.pod=prometheus-stack-grafana-7594dc6d95-cjfjj container=87d81d268f9c image=quay.io/kiwigrid/k8s-sidecar:1.22.0 connection=10.244.2.25:35680->10.0.0.1:443)';
+        console.log(sampleJson);
```
Comment (jacobbeasley) [inline dash/backend/src/modules/auth/controllers/non-redirectable-login.controller.ts:112]: Is this lefta round by accident?

File: dash/backend/src/util/tablify.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+export function tablify(obj: any, depth = 0, maxdepth = 10) {
```
Comment (becky-intelletive) [inline dash/backend/src/util/tablify.ts:1]: Where is this function from?
Wondering if we should c+p the Tablify class from the frontend. I did a bit of bug fixing on it :)

**Review by becky-intelletive — CHANGES_REQUESTED** [review]

File: dash/backend/src/util/tablify.ts (hunk 2 above)
Comment (jacobbeasley) [reply dash/backend/src/util/tablify.ts:1]: It looks a bit similar to this - https://github.com/m9sweeper/m9sweeper/blob/974365e590265f452b62c1b00facf1b94820d79a/dash/frontend/src/app/core/pipe/tablify.pipe.ts#L6

Comment (jacobbeasley) [reply dash/backend/src/util/tablify.ts:1]: Though I don't think we want the backend to import PipeTransform, so we wouldn't copy paste the whole thing - just parts of it (maybe that's what Charis did). @becky-intelletive  Do you see a specific change required here?

Comment (becky-intelletive) [reply dash/backend/src/util/tablify.ts:1]: You're right. On closer inspection it looks like it was a c+p :)

File: dash/backend/src/modules/auth/controllers/non-redirectable-login.controller.ts (hunk 1 above)
Comment (charisprose) [reply dash/backend/src/modules/auth/controllers/non-redirectable-login.controller.ts:112]: I gave it a test after I added the function. Might have missed to remove them. The lines will be removed.

File: dash/backend/src/util/tablify.ts (hunk 2 above)
Comment (charisprose) [reply dash/backend/src/util/tablify.ts:1]: @jacobbeasley The frontend pipe was from Angular. Per the instruction from the relevant ticket, this was modified a bit to provide the same usage. Let me know if you want any further changes.

**Review by becky-intelletive — DISMISSED** [review]

**Review by becky-intelletive — APPROVED** [review]

File: dash/backend/src/email-templates/error-email.hbs [hunk 3]
```diff
@@ -30,7 +30,7 @@
         </p>
 
         <p>
-            {{{data}}}
+            {{data}}
```
Comment (kyle-intelletive) [inline dash/backend/src/email-templates/error-email.hbs:33]: Did you change from using something besides handlebars?

Handlebars does use triple brackets

Comment (becky-intelletive) [reply dash/backend/src/email-templates/error-email.hbs:33]: Good catch!

**Review by becky-intelletive — APPROVED** [review]

**Pull request merged**